helix-core/src/main/java/org/apache/helix/task/JobConfig.java

```diff
@@ -449,11 +446,6 @@ public Builder setMaxAttemptsPerTask(int v) {
       return this;
     }
 
-    public Builder setMaxForcedReassignmentsPerTask(int v) {
```

Comment:
same here, do not remove the method.  Deprecate it and clearly comment that this field will no longer used by Helix.